unittests/src/sinks/console-json-sink-tests.cpp

```diff
@@ -0,0 +1,225 @@
+/**
+ * @file console-json-sink-tests.cpp
+ * @author Denis Sheyer (denis.sheyer@gmail.com)
+ * @brief
+ * @date 12025-01-16
+ *
+ * @copyright Copyright (c) 2025
+ *
+ */
+
+#include <catch2/catch_all.hpp>
+
+#include "catch2-matchers.hpp"
+#include "log-mock.hpp"
+#include "logger-mock.hpp"
+#include "octo-logger-cpp/channel.hpp"
+#include "octo-logger-cpp/context-info.hpp"
+#include "octo-logger-cpp/log-level.hpp"
+#include "octo-logger-cpp/log.hpp"
```

Comment:
Shouldn't `log` come before `log-level`?

```suggestion
#include "octo-logger-cpp/log.hpp"
#include "octo-logger-cpp/log-level.hpp"
```

Reply:
https://github.com/ofiriluz/octo-logger-cpp/pull/32#discussion_r1921879925